Kentico/cloud-sample-app-react
# Update from original (#2)

* Visitor tracking

App extended with tracking code and a custom activity call. Updated README. Needs testing and proofreading.

* Visitor tracking - tested by author

Missing onClick event found and added.

* IMHO it can go public

Few typos fixed in README.

* Both issues fixed

Tracking key should be the same everywhere. Logging functionality has been separated to its own file in Utilities.

* Fixes after the review

Tracking key should be the same everywhere. Logging functionality has been separated to its own file in Utilities.

* Ignore this one

* Both issues fixed

Tracking key should be the same everywhere. Logging functionality has been separated to its own file in Utilities. Wording update of Readme.

* Fine tuning

- Cleanup to do not overwrite sample data automatically.
- Rename of the logging function to follow best practices.
- Readme update.

* Update README.md

* content items dump

* Added exported content items for translation into Spanish. DC-306

* Added remaining files for translation. DC-306

* Spanish Translations

Translated content of "name" fields to Spanish.

* Add Spanish for "Sydney"

Duplicated code block, added comma after last original curly bracket, added copied block of code, changed the value of "city" in the copied block of code and changed the language id to 028f0e05-9f17-45be-962d-62bebe6b6f53. Wrapped everything in brackets. 

Unsure about format / indention / alignment. 

Waiting for feedback to continue with the rest of the files.

* Fixed language Id

Changed 028f0e05-9f17-45be-962d-62bebe6b6f53 to d1f95fde-af02-b3b5-bd9e-f232311ccab8 in the language field of the code block that contains the translation.

* Change "y" for "e", adjusting title

Title:

If the "on" of "On Roasts" means "about", a better translation would be "Sobre".
"Rostizado" is used more as a degree of roasting than for roasting itself, that's why I suggest to use "Tueste". This new title could translate to "About the action of roasting"

Summary / description:

In Spanish we generally use "y" for "and", except when the second word begins with a vowel or h (because our h is silent).

Notes:

The languages are the other way around here. What's the convention you prefer? 
Should I also change the keywords in Spanish? (to tueste, probably?)

* Update README.md

* included "tueste" as meta_keyword

"tueste" now substitutes "asados" in the meta_keyword field to match the translation in those of title, summary, and meta_description.

* Tile fix

From "Café tostado de primera calidad" to "Café tostado premium". 

Just to be clear, the "premium" in the translation applies to the coffee once it is already roasted, not to the roasting process.

* Add Spanish translations

Added block for translation, changed language id, translated relevant content - "city" (Ámsterdam) and "country" (Países Bajos).

* Added block

No translation required.
Added code block and included language id.

* Fixed spelling

Á instead of A in "Los Ángeles".

* Spelling

"Sídney" instead of "Sydney".

* Translated city and country field content

London, United Kingdom = Londres, Reino Unido

* Translated country field content

Spain = España

* Add Spanish translation

Vacuum pot = Cafetera de vacío

* Add Spanish translation

Hand grinder = Molinillo de mano

* Spanish translation for East

Oriental = From the East side, on the East side.

Alternatively, "Melbourne Este", "Melbourne del Este" can do.

* Duplicated block, added language ID

No translation needed.

* Add Spanish Translation

Paper filters = Filtros de papel

* Add Spanish translations

Filters = Filtros

* Updated License Time period.

You should consider having a php script that auto detects the license's year and changes it accordingly.

* Fixed translation

Minor style revising.

* Fixed translation

Minor style revising

* Optional Fix

New York is also referred as Nueva York, but it can be understood either way.

* Fixed translation

Some spelling and style translations.

You can get rid of "a la vida" if you want it to say "it has come back" instead of "it has been brought back to life".

* Spanish Translation

Brazil = Brasil (in "product name" and "country")

Also translated 'yellow' in "variety"

* Fixed translation

Changed "Sede" for "Oficina Central"

* No translation needed

Duplicated block of code, added language id

* No translation needed

Duplicated block of code, added language id
